src/components/Asset/AssetActions/ButtonBuy/index.tsx

```diff
@@ -86,7 +90,9 @@ function getAlgoHelpText(
       ? `Connect to the correct network to interact with this asset.`
       : isBalanceSufficient === false
       ? ''
-      : `Additionally, you will buy 1 ${dtSymbolSelectedComputeAsset} for the ${selectedComputeAssetType} and spend it back to its publisher and pool.`
+      : algorithmPriceType === 'free'
+      ? `Additionally, the selected ${selectedComputeAssetType} is free to use.`
```

Comment:
shouldn't we mention about the fees on free algorithms as well, since we mention about the fees for the the datasets also ?

Reply:
It would probably be best if we had one sentence that covered both, to save repeating ourselves. I'll adjust this